packages/create-sitecore-jss/src/templates/angular/scripts/install-pre-push-hook.ts

```diff
@@ -1,29 +1,34 @@
 ﻿import * as fs from 'fs';
+import * as path from 'path';
+import { promisify } from 'util';
+import { exec } from 'child_process';
 
-const installHooks = () => {
-  // data to be written to the file
-  const data = `#!/bin/sh
-#
-# pre-push hook runs our linter before we push our changes
-#
-# To skip this hook, use the --no-verify flag
-# when pushing.
-#
+const gitRootDirMessage = '\x1b[32m%s\x1b[0m';
+const successMessage = '\x1b[32m%s\x1b[0m';
+const errorMessage = '\x1b[31m%o\x1b[0m';
 
-echo "Running lint check..."
-npm run lint`;
+const installHooks = async () => {
+  try {
+    const appPath = path.join(__dirname, '..').replace(/\\/g, '/');
+    const { stdout } = await promisify(exec)('git rev-parse --show-toplevel');
+    const gitRootPath = stdout.trim();
+    console.log(gitRootDirMessage, 'Writing data to local .git folder...');
 
-  // \x1b[32m%s\x1b[0m - set color to green, insert the string, reset color after string is logged to console
-  console.log('\x1b[32m%s\x1b[0m', 'Writing to local .git folder...');
+    const data = `#!/bin/sh
+    # pre-push hook runs our linter before we push our changes
+    #
+    # To skip this hook, use the --no-verify flag
+    # when pushing.
+    echo "Running lint check..."
+    cd ${appPath}
+    npm run lint
+  `;
```

Comment:
Do we need to remove indentation here (so that it does not show up in resulting file)?